tests/e2e/test_browser.py

```
@@ -314,7 +315,7 @@ def test_server_health(base_url: str) -> None:
     """Verify server responds to health check before running browser tests."""
     import urllib.request
     try:
-        r = urllib.request.urlopen(f"{base_url}/api/ping", timeout=10)
+        r = urllib.request.urlopen(f"{base_url}/api/health", timeout=10)
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)